classes/security/authorization/OjsIssueRequiredPolicy.inc.php

```diff
@@ -64,6 +64,31 @@ function dataObjectEffect() {
 		$this->addAuthorizedContextObject(ASSOC_TYPE_ISSUE, $issue);
 		return AUTHORIZATION_PERMIT;
 	}
+
+	/**
+	 * @copydoc DataObjectRequiredPolicy::getDataObjectId()
+	 * Considers a not numeric public URL identifier
+	 */
+	function getDataObjectId() {
+		// Identify the data object id.
+		$router = $this->_request->getRouter();
+		switch(true) {
+			case is_a($router, 'PKPPageRouter'):
+				if ( is_numeric($this->_request->getUserVar($this->_parameterName)) ) {
```

Comment:
Might be better to use `ctype_digit` as in [this example from the stable codebase](https://github.com/pkp/ojs/blob/ojs-stable-2_4_8/classes/issue/IssueDAO.inc.php#L183). `is_numeric` will return true with e.g. `0xa`, `2e3`, and other unexpected notations.